jaeger-apachehttpclient/src/test/java/com/uber/jaeger/httpclient/JaegerRequestAndResponseInterceptorIntegrationTest.java

```diff
@@ -78,7 +77,9 @@ public void setUp() {
 
   @After
   public void tearDown() throws Exception {
-    TestUtils.resetGlobalTracer();
+    Field field = TracingUtils.class.getDeclaredField("tracer");
+    field.setAccessible(true);
+    field.set(null, null);
```

Comment:
This clean up is repeated in a few places, could you make move it to `TestUtils` and reuse?

Reply:
Actually becoming a bit of a dep mess when I do that since `TestUtils` is a part of the core module